gnocchi/opts.py

```diff
@@ -218,6 +218,32 @@ def list_opts():
                 default=10,
                 help='Delay between flushes'),
         )),
+        ("amqp1d", (
+            cfg.HostAddressOpt('host',
+                               default='localhost',
+                               required=True,
+                               help='The listener IP for amqp1.0 '),
+            cfg.PortOpt('port',
+                        default=5672,
```

Comment:
If you don't set `required=True`, what happens here? I guess it could be `None` and then it'll fail.